### ISSUE TITLE
edit button in settings.json only works with right clicking

### ISSUE THREAD

**Issue by imba-tjd** [microsoft/vscode#75026]
From https://github.com/microsoft/vscode/issues/73977 but he wasn't willing to open his issue.

Left click is broken but right click works. It's introduced in 1.34.0. I can reproduce it by upgrading from 1.33.1 to 1.34.0.

@roblourens Is it by designed that only right click works?

- VSCode Version: 1.34.0 and 1.35.0
- OS Version: Windows 10 1903 and Windows 7 and Debian 10

Steps to Reproduce: The same as https://github.com/microsoft/vscode/issues/73977.

Does this issue occur when all extensions are disabled?: Yes

<details>

```
Version: 1.34.0 (user setup)
Commit: a622c65b2c713c890fcf4fbf07cf34049d5fe758
Date: 2019-05-15T21:59:53.881Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT ia32 6.1.7601

CPUs	Intel(R) Core(TM) i7-4710MQ CPU @ 2.50GHz (4 x 2494)
GPU Status	2d_canvas: enabled
checker_imaging: disabled_off
flash_3d: enabled
flash_stage3d: enabled
flash_stage3d_baseline: enabled
gpu_compositing: enabled
multiple_raster_threads: enabled_on
native_gpu_memory_buffers: disabled_software
rasterization: unavailable_off
surface_synchronization: enabled_on
video_decode: enabled
webgl: enabled
webgl2: enabled
Load (avg)	undefined
Memory (System)	2.00GB (1.01GB free)
Process Argv	
Screen Reader	no
VM	33%
```
</details>

**Comment by roblourens** [comment]
Now I can reproduce it. You can disable custom menus with `window.titleBarStyle` to work around it if that helps

**Comment by imba-tjd** [comment]
Yes, `"window.titleBarStyle": "native"` takes effects.

**Comment by jeanp413** [comment]
I pushed a PR #75967 to fix this but in a duplicated issue #75947 (just found this one sorry about the duplicate)

**Comment by imba-tjd** [comment]
Tried the latest insider version, works.
